### PR TITLE
[Docs] Release notes for PWA Studio release 12.7

### PR DESCRIPTION
**Pull request by bdenham**
## Description

Release notes for PWA Studio release 12.7

## Completion Blocker

We need to include the new Lighthouse scores. The included ones are from the 12.6 release.

## Related Issue

[PWA-3000](https://jira.corp.adobe.com/browse/PWA-3000)

Closes #PWA-3000.

## Acceptance

- @anthoula 
- @dpatil-magento 

### Verification Stakeholders

- @anthoula 
- @dpatil-magento 

## Verification Steps

Review CHANGELOG.md to ensure all 12.7 stories and bugs are added and described accurately.

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -57,46 +68,47 @@ See [Upgrading versions][] for more information about upgrading between PWA Stud
...
-| `venia-sample-language-packs`       | **0.0.10**     |
-| `venia-sample-payments-checkmo`     | **0.0.8**      |
-| `pagebuilder`                       | **7.5.0**      |
-| `peregrine`                         | **12.5.0**     |
-| `pwa-buildpack`                     | **11.4.0**     |
+| `babel-preset-peregrine`            | **1.2.2**      |
+| `create-pwa`                        | **2.3.3**      |
+| `experience-platform-connector`     | **1.0.2**      |
+| `upward-security-headers`           | **1.0.11**     |
+| `venia-sample-backends`             | 0.0.8      |
```
Comment (anthoula) [inline CHANGELOG.md:82]: venia-sample-backends is bumped to 0.0.9 since dependent pwa-buildpack changed

**Review by anthoula — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
--   Accessibility Improvements: Keyboard focus no longer lands on hidden elements in Kebab menu — [3800][]   
--   Fixed a bug in mobile view where the Edit and Remove options in the kebab menu for a cart item on the cart page is non functional. - [3925][]
--   Fixed a console error when accessing different filters. - [30][]
+The 12.7.0 release of PWA Studio focuses on accessibility improvements and bug fixes.
 
-## 12.6.0 Lighthouse scores
+### Features
 
-With each new release of PWA Studio, we perform Lighthouse audits of four Venia page types, each representing a different level of complexity. Shown below are the Lighthouse scores for the 12.6.0 release of these pages on desktop and mobile devices.
+- **Out-of-stock for product variations** [3903][] — The out-of-stock setting can now be applied to configurable product variations, making it easier for customers to select variations.
```
Comment (dpatil-magento) [inline CHANGELOG.md:13]: Should this [pr](https://github.com/magento/pwa-studio/pull/3903/) be part of Highlights as well? It was a pretty big pr with good improvements.

**Review by dpatil-magento — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bdenham) [reply CHANGELOG.md:13]: Yeah, for sure. I missed that one for some reason. I'll add it in first thing in the morning. 👍

Comment (bdenham) [reply CHANGELOG.md:13]: @dpatil-magento, After following your pr link, it looks like the same thing as the feature mentioned: **Out-of-stock for product variations**. Is there another related feature that I'm missing?

Comment (bdenham) [reply CHANGELOG.md]: Never mind. I get it. Fixed.

**Comment by dpatil-magento** [comment]
run pr-test

**Pull request merged**